nix/test/mayastorLib/MayastorUtils.py

```diff
@@ -0,0 +1,11 @@
+DEFAULT_NEXUS_PORT = "8420"
+DEFAULT_REPLICA_PORT = "8430"
+
+def subsystem_is_discoverable(host, ip, port, subsys):
```

Comment:
Maybe prefix function name with "nvme" as this is nvme specific. We could possibly want to discover other subsystems in the future.

Reply:
Even better, I moved it out into an nvmfUtils file